src/components/SummaryPanel/RpsChart.tsx

```diff
@@ -52,11 +52,12 @@ const renderNoTrafficLegend = () => {
   );
 };
 
-const thereIsTrafficData = serieData => {
+const thereIsTrafficData = seriesData => {
   return (
-    serieData.length > 0 &&
-    serieData[0].length > 1 &&
-    serieData[1].slice(1).reduce((accum, val) => accum + parseFloat(val), 0) > 0
+    seriesData &&
+    seriesData.length > 1 &&
+    seriesData[0].length > 1 &&
+    seriesData[1].slice(1).reduce((accum, val) => accum + parseFloat(val), 0) > 0
```

Comment:
Can we use the typescript `Number(123)` instead of `parseFloat(123)`. Or even the unary operator `+` although that is easy to miss. `parseInt` and `parseFloat` are antipatterns in typescript.

https://stackoverflow.com/questions/14667713/typescript-converting-a-string-to-a-number

Reply:
`Number()` is much more typesafe: `Number('123abc')`returns `NaN` while `parseFloat('123abc')` returns `123`

Reply:
Fine by me, I updated it.